sorc/ncep_post.fd/ALLOCATE_ALL.f

```diff
@@ -70,6 +72,44 @@ SUBROUTINE ALLOCATE_ALL()
       allocate(train(im,jsta_2l:jend_2u,lm))
       allocate(tcucn(im,jsta_2l:jend_2u,lm))
       allocate(EL_PBL(im,jsta_2l:jend_2u,lm))
+
+!Initialization
```

Comment:
Can you add threading in the variable allocation loops? I think POST thread safe now. Other changes look good to me.